cli/popper/dry.py

```diff
@@ -0,0 +1,78 @@
+import os
```

Comment:
most of this logic is already part of the `ActionRunner` and its descendants. In order to avoid duplicating logic, it would be better to have action runners get instructed whether they are printing what they'd do. This can follow the same approach than the verbose feature (PR #527). That is, adding a new member of `ActionRunner` (e.g. a `self.verbose` attribute) and extending the constructor so that this is received when instantiating runners. This also needs to be part of a Workflow's constructor